dap-python.el

```diff
@@ -238,7 +245,12 @@ overriden in individual `dap-python' launch configurations."
          (cl-remf conf :module))
        (unless (plist-get conf :cwd)
          (cl-remf conf :cwd))
-
+       (when (and debug-options
+                  (sequencep debug-options)
+                  (seq-contains debug-options "DebugStdLib")
+                  (null justmycode))
+         (warn "DebugStdLib is deprecated, use justMyCode, please consider to update your configuration.")
```

Comment:
Why the backwards compatibility considerations here? `:debugOptions` is introduced with this PR, so there cannot be a config in the wild using it.

Reply:
I'd just drop that warning altogether. You could state in the README somewhere that :debugOptions is `ptsvd`-specific, and that `:justMyCode` is the preferred way to go about this, since it is portable.